app/bundles/PageBundle/Entity/PageRepository.php

```diff
@@ -23,9 +23,15 @@ class PageRepository extends CommonRepository
      */
     public function getEntities(array $args = [])
     {
+        //use a subquery to get a count of submissions otherwise doctrine will not pull all of the results
+        $sq = $this->_em->createQueryBuilder()
+            ->select('count(fs.id)')
+            ->from('MauticFormBundle:Submission', 'fs')
+            ->where('fs.page = p');
+
         $q = $this
             ->createQueryBuilder('p')
-            ->select('p')
+            ->select('p, ('.$sq->getDql().') as submission_count')
```

Comment:
This wasn't very good idea because the `getEntities()` method is used from many places. Now all of those places call the query to the form_submissions table which is unnecessary and also broke global search https://github.com/mautic/mautic/pull/11435 :/

Reply:
You are right, my mistake @escopecz :/

Reply:
@alfredoct96 are you able to take this on? Just trying to bring this back into attention :)

Reply:
@alfredoct96 are you still here?

Reply:
Hi, unfortunately these days I'm busy with work

Reply:
Fixed in https://github.com/mautic/mautic/pull/11842. Please review/test.